ledfx/devices/rpi_ws281x.py

```
@@ -62,63 +70,58 @@ def __init__(self, ledfx, config):
-            for led in len(self.pixel_count):
-                self.strip.setPixelColor(
-                    led,
-                    self.buffer[led],
-                    self.buffer[led + 1],
-                    self.buffer[led + 2],
-                )
-            self.strip.show()
-
-    def swap(self, array, pos1, pos2):
-        array[pos1], array[pos2] = array[pos2], array[pos1]
+            pixel += 1
```

Ruff

Use `enumerate()` for index variable `pixel` in `for` loop

(SIM113)